tests/integration/hive_metastore/test_migrate.py

```diff
@@ -134,7 +138,11 @@ def test_migrate_external_table(
     make_mounted_location,
 ):
     src_schema = runtime_ctx.make_schema(catalog_name="hive_metastore")
-    src_external_table = runtime_ctx.make_table(schema_name=src_schema.name, external_csv=make_mounted_location)
+    src_external_table = runtime_ctx.make_table(
+        schema_name=src_schema.name,
+        external_csv=make_mounted_location,
+        columns=[ColumnInfo(name="`backtick", type_name=ColumnTypeName.STRING)],  # Test with column that needs escaping
```

Comment:
The backtick (<code>&#x60;</code>) needs to be in the middle of the identifier, please. 😀 At either end I don't fully trust the escaper not to be stripping it.